### PR TITLE
WIP: use systemd to manage ceph daemons

### PR DESCRIPTION
**Pull request by rootfs**
mon is now started by systemd on Atomic.

### CODE REVIEW

File: roles/ceph-mon/tasks/docker/main.yml [hunk 1]
```diff
@@ -5,6 +5,14 @@
   changed_when: false
   failed_when: false
 
+- name: check if it is Atomic host
+  stat: path=/run/ostree-booted
```
Comment (leseb) [inline roles/ceph-mon/tasks/docker/main.yml]: Please use this syntax:
```
stat:
  path: /run/ostree-booted
```

File: roles/ceph-mon/tasks/docker/start_docker_monitor.yml [hunk 2]
```diff
@@ -10,6 +10,43 @@
 - name: pull ceph daemon image
   shell: "docker pull {{ ceph_mon_docker_username }}/{{ ceph_mon_docker_imagename }}"
 
+# Use systemd to manage container on Atomic host
+- name: generate systemd unit file
+  sudo: true
+  config_template:
```
Comment (leseb) [inline roles/ceph-mon/tasks/docker/start_docker_monitor.yml]: I don't think `config_template` is needed here (unless you want users to be able to add more variables to the unit file).
Simply use the `template` module.

File: roles/ceph-mon/meta/main.yml [hunk 3]
```diff
@@ -11,4 +11,4 @@ galaxy_info:
   categories:
     - system
 dependencies:
-  - { role: ceph-common, when: not mon_containerized_deployment }
```
Comment (leseb) [inline roles/ceph-mon/meta/main.yml]: maybe it's part of your previous patches but why do we need to include `ceph-common` role for docker?

**Comment by rootfs** [comment]
we should label containers and use `atomic` command to run and install containers. Will open another PR for Atomic.

**Pull request closed without merging**